### DISCUSSION TITLE
It's possible to set primary color blue??

### DISCUSSION THREAD

**Discussion by j-vitali** [themesberg/flowbite-react#1406] [Q&A]
It's possible to set primary color blue??
Or I should write it manually for every component...

**Comment by SutuSebastian** [comment]
There is work planned to do this right in `tailwind.config.js`, and the `flowbite.plugin()` will setup this for u automatically setting the default primary blue colors of https://flowbite.com/.

**Comment by SutuSebastian** [comment]
Fixed by [flowbite-react@0.11.0](https://github.com/themesberg/flowbite-react/releases/tag/flowbite-react%400.11.0) release.

See https://flowbite-react.com/docs/customize/colors#customizing-colors.
